### PR TITLE
Add company

### PR DESCRIPTION
**Pull request by simonvanlierde**
Adding Metabolic to the company list

- [x] I read [CONTRIBUTING.md](https://github.com/pogopaule/awesome-sustainability-jobs/blob/main/CONTRIBUTING.md)  
- [x] I didn't change README.md (but instead src/data.yaml or src/template.md)

### CODE REVIEW

File: src/data.yaml [hunk 1]
```diff
@@ -1,4 +1,30 @@
...
+    jobs: https://www.metabolic.nl
+    description: Sustainability consulting
+    remote: true
+    speculative: false
+    field: Misc.
+    geo:
+      - country: Netherlands
+        lat: 52.399131
+        long: 4.909506
+    review: https://www.glassdoor.com/Overview/Working-at-Metabolic-EI_IE1257662.11,20.htm
```
Comment (pogopaule) [inline src/data.yaml]: This URL needs to point to the Reviews page, otherwise the crawler can't find the rating.

File: src/data.yaml [hunk 2]
```diff
@@ -1,4 +1,30 @@
...
+    rating: '4.4'
```
Comment (pogopaule) [inline src/data.yaml]:
```suggestion
    rating: 4.4
```
needs to be a number without quotes

File: src/data.yaml [hunk 3]
```diff
@@ -1,4 +1,30 @@
...
+    description: Heat pump systems
+    remote: true
+    speculative: false
+    field: Renewable Energies
+    geo:
+      - country: USA
+        lat: 40.6311127
+        long: -73.9464322
+    review: https://www.glassdoor.com/Reviews/BlocPower-Reviews-E1453559.htm
+    rating: '4.2'
```
Comment (pogopaule) [inline src/data.yaml]:
```suggestion
```
BlocPower is already present. See entry right below.

File: src/data.yaml [hunk 4]
```diff
@@ -1,4 +1,30 @@
 jobs:
+- name: Metabolic
+    website: https://www.metabolic.nl
```
Comment (pogopaule) [inline src/data.yaml:3]: This and the following lines needs to align with `name`.

**Review by pogopaule — CHANGES_REQUESTED** [review]
Hey @camobs! Thank you very much for your contribution! :+1: 
There are some minor changes you need to push before I can merge.

**Comment by simonvanlierde** [comment]
Hi pogopaule!
Thanks for your review, I've pushed your recommended changes to the branch!

**Pull request merged**